<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>slitunit</class>
 <widget class="QWidget" name="slitunit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>737</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>90</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>200</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_0_AKTION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>260</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>20</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>50</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:ES_0_AKTION</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>120</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_GET</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>290</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_2_AKTION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>170</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>More</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_26">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>^</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:ES_0_AKTION</string>
   </property>
  </widget>
  <widget class="QCAPushButton" name="pushButton_27">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>v</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:ES_0_AKTION</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_41">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_0_GET</string>
   </property>
  </widget>
  <widget class="QCALabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>59</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_0_Name</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>590</y>
     <width>981</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>660</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis_1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>VDSO</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:ES_0_SW</string>
   </property>
   <property name="units" stdset="0">
    <string>µm</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:ES_0_Input</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>HDSO</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:PH_0_GET</string>
   </property>
   <property name="units" stdset="0">
    <string>µm</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_0_SET</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis_3" native="true">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>449</width>
     <height>31</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>HDST</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:PH_1_GET</string>
   </property>
   <property name="units" stdset="0">
    <string>µm</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_SET</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis_4" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>SUT</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:PH_2_GET</string>
   </property>
   <property name="units" stdset="0">
    <string>µm</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_2_SET</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis_5" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>Speed</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:ES_0_JOGSPEED</string>
   </property>
   <property name="units" stdset="0">
    <string>mm/s</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:ES_0_SETJOGSPEED</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis_6" native="true">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>650</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>Low Level Motor</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:ES_0_JOGSPEED</string>
   </property>
   <property name="units" stdset="0">
    <string>mm/s</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:ES_0_SETJOGSPEED</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis_7" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>400</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>HDST</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:PH_1_GET</string>
   </property>
   <property name="units" stdset="0">
    <string>um</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_1_SET</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis_8" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>VDSO</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:ES_0_SW</string>
   </property>
   <property name="units" stdset="0">
    <string>um</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:ES_0_Input</string>
   </property>
  </widget>
  <widget class="QCAAxis" name="Axis_9" native="true">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>540</y>
     <width>449</width>
     <height>32</height>
    </rect>
   </property>
   <property name="description" stdset="0">
    <string>SUT</string>
   </property>
   <property name="pvReadback" stdset="0">
    <string>u411pgm1:PH_2_GET</string>
   </property>
   <property name="units" stdset="0">
    <string>um</string>
   </property>
   <property name="pv" stdset="0">
    <string>u411pgm1:PH_2_SET</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCALabel</class>
   <extends>QLabel</extends>
   <header>qcalabel.h</header>
  </customwidget>
  <customwidget>
   <class>QCAAxis</class>
   <extends>QWidget</extends>
   <header>qcaaxis.h</header>
  </customwidget>
  <customwidget>
   <class>QCAPushButton</class>
   <extends>QPushButton</extends>
   <header>qcapushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
